ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemRootResourceDefinition.java

```diff
@@ -308,19 +308,20 @@ private static void registerTransformers1_2_1_and_1_3_0(SubsystemRegistration su
         builder.rejectChildResource(PathElement.pathElement(EJB3SubsystemModel.REMOTING_PROFILE));
         if (version.equals(VERSION_1_2_1)) {
             TimerServiceResourceDefinition.registerTransformers_1_2_0(builder);
+            EJB3RemoteResourceDefinition.registerTransformers_1_2_0(builder);
+            MdbDeliveryGroupResourceDefinition.registerTransformers_1_2_0(builder);
```

Comment:
I think this one should also get called for 1.3.0? In which case the method on MdbDeliveryGroupResourceDefinition could be renamed